### PR TITLE
feat: app details cards devtron & helm apps

### LINKED ISSUE #1452: New App Details Cards on App Details
**Problem To Be Solved**
Revamp of the the cards displaying different kind of information on the App Details info for the Devtron apps.

### PR DESCRIPTION
**Pull request by prajjwal-devtron**
# Description

Fixes #1452

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
- [ ] Manually tested all the functionalities of these cards

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -3457,7 +3457,7 @@ textarea,
 }
 
 .dc__flip-90 {
-    transform: rotate(-90deg);
+    transform: rotate(90deg);
```
Comment (vivek-devtron) [inline src/css/base.scss:3460]: is it intentional? It has been used on multiple places

File: src/components/app/details/appDetails/DeployedCommitCard.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                            placement="top"
+                            content="Status of last triggered deployment" // @TODO: update this copy
+                        >
+                            <Question className="icon-dim-16 mt-2" />
+                        </Tippy>
+                    </div>
+                    <div className="app-details-info-card__top-container__content__commit-text-wrapper flex fs-12 fw-4">
+                        <CommitIcon className="app-details-info-card__top-container__content__commit-text-wrapper__commit-icon" />
+                        <div className="app-details-info-card__top-container__content__commit-text-wrapper__commit-sha">
+                            574588a3
```
Comment (vivek-devtron) [inline src/components/app/details/appDetails/DeployedCommitCard.tsx:26]: Why is it hardcoded?

Comment (prajjwal-devtron) [reply src/components/app/details/appDetails/DeployedCommitCard.tsx:26]: Data was not integrated then, it has integrated now as part of the last commit of this pr

File: src/css/base.scss (hunk 1 above)
Comment (prajjwal-devtron) [reply src/css/base.scss:3460]: Yes, there were duplicate styles of 270 deg and -90deg, have used 270 deg style wherever the -90 deg was needed. And created this 90 deg rotation for an icon used in `Deployment status` card

File: src/components/app/details/appDetails/DeployedCommitCard.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    loadingResourceTree,
+    showCommitInfoDrawer,
+    envId,
+    ciArtifactId,
+}: DeployedCommitCardType) => {
+    const [commitId, setCommitId] = useState<string>(null)
+    const [commitMessage, setCommitMessage] = useState<string>(null)
+
+    useEffect(() => {
+        let params = {
```
Comment (vivek-devtron) [inline src/components/app/details/appDetails/DeployedCommitCard.tsx:20]: please make it const. we can directly pass this instead of creating a variable

**Review by vivek-devtron — DISMISSED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**